info.csv

```diff
@@ -19,3 +19,4 @@ Himanshu Singh,          17SCSE101209,           1713101199,            None,
 PRIYANSHU TRIVEDI,       17SCSE105051,           1713105051,            NONE,                            2nd,         1-cloud,  C C++ PYTHON,                 competitive coding,        NA,                  trivedipriyanshu972@gmail.com,    https://github.com/PRIYANSHU-AMBITION,   https://www.linkedin.com/in/priyanshu-trivedi-54020b189/,  NA
 Ankit Singh,             17SCSE114059,           1713114009,            Web Development,                 2,           1,        "python and HTML CSS",        Web development and AI,   NA,                   ankitsingh4301@gmail.com,         https://github.com/1AnkitSingh02,       https://www.linkedin.com/in/ankit-singh-1b968a170,          NA
 ketan singh,             17SCSE101558,           1713101536,            Web development,                 2,           2,         NA,                          web development,            NA,                  ketansingh343@gmail.com,         https://github.com/KeTan05,              https://www.linkedin.com/in/ketan-singh-a6093816b/,        NA
+Piyush Kumar,            18SCSE1120015,          18021120014,           Machine Learning,               1,            9,        C++,                          "CLOUD COMPUTING",        NA,                   pkumar22122001@gmail.com       `   https://github.com/kpiyush04,        https://www.linkedin.com/in/piyush-kumar-561309156/ ,    living on world of clouds
```

Comment:
You forgot a ` , ` after your email address!

Reply:
changes updated